android/sdl_android/src/main/java/com/smartdevicelink/proxy/SdlProxyBase.java

```diff
@@ -2093,7 +2100,66 @@ private void handleErrorsFromInternalMessageDispatcher(String info, Exception e)
 		_proxyListener.onError("Proxy callback dispatcher is down. Proxy instance is invalid.", e);
 	}
 	/************* END Functions used by the Message Dispatching Queues ****************/
-	

```

Comment:
Did the spacing change causing a delete and add?

Reply:
Adding new code right after the line (new line character) causes the editor to make the line delete/add.